app/src/main/java/com/mapbox/maps/testapp/examples/DebugModeActivity.kt

```diff
@@ -53,6 +54,13 @@ class DebugModeActivity : AppCompatActivity() {
     mapboxMap.subscribe(observable, listOf(MapEvents.RESOURCE_REQUEST))
     // Using the extension method
     mapboxMap.subscribeResourceRequest(extensionObservable)
+    mapboxMap.loadStyle(style("") {})
```

Comment:
do we really need such setup?

Reply:
Nooo, that remainder from test setup, will remove